lib/Guacamole.pm

```diff
@@ -355,6 +355,7 @@ GlobalVarExpr ::= '$#'
                 | SigilArray  GlobalVariables ElemSeq0
                 | SigilHash   GlobalVariables ElemSeq0
                 | SigilGlob   GlobalVariables ElemSeq0
+                | SigilArrayTop GlobalVariables ElemSeq0
```

Comment:
I don't think array top allows any element access operators to follow the identifier, so `ElemSeq0` can be removed.

Reply:
Updated the commit.  Thanks for the advice